notebooks/00_quick_start/rlrmc_movielens.ipynb

```diff
@@ -0,0 +1,267 @@
+    "\n",
+    "# rank of the model, a positive integer (usually small), required parameter\n",
+    "rank_parameter = 10\n",
+    "# regularization parameter multiplied to loss function, a positive number (usually small), required parameter\n",
+    "regularization_parameter = 0.001\n",
+    "# initialization option for the model, 'svd' employs singular value decomposition, optional parameter\n",
+    "initialization_flag = 'svd' #default is 'random'\n",
+    "# maximum number of iterations for the solver, a positive integer, optional parameter\n",
+    "maximum_iteration = 10 #optional, default is 100\n",
+    "# maximum time in seconds for the solver, a positive integer, optional parameter\n",
+    "maximum_time = 300#optional, default is 1000"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = RLRMCalgorithm(rank = rank_parameter,\n",
+    "                       C = regularization_parameter,\n",
+    "                       model_param = data.model_param,\n",
+    "                       initialize_flag = initialization_flag,\n",
+    "                       maxiter=maximum_iteration,\n",
+    "                       max_time=maximum_time)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Verbosity of the intermediate results\n",
+    "verbosity=0 #optional parameter, valid values are 0,1,2, default is 0\n",
+    "# Whether to compute per iteration train RMSE (and test RMSE, if test data is given)\n",
+    "compute_iter_rmse=True #optional parameter, boolean value, default is False"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Hyper-parameters of the algorithm\n",
+      "Rank: 10, Regularization parameter: 1.000000e-03\n",
+      "Took 6.238836765289307 seconds for training.\n"
+     ]
+    }
+   ],
+   "source": [
+    "start_time = time.time()\n",
+    "\n",
+    "model.fit(data,verbosity=verbosity,compute_iter_rmse=compute_iter_rmse)\n",
+    "\n",
+    "train_time = time.time() - start_time\n",
+    "\n",
+    "print(\"Took {} seconds for training.\".format(train_time))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Obtain predictions from the RLRMC model on the test data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Obtain predictions on (userID,itemID) pairs (60586,54775) and (52681,36519)\n",
+    "output = model.predict([60586,52681],[54775,36519])\n",
+    "# Obtain prediction on the full test set\n",
+    "output_full = model.predict(test['userID'].values,test['itemID'].values)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Reference\n",
```

Comment:
I recall we want to use a different term, wast that "Further readings" @miguelgfierro @wutaomsft ?

Reply:
there is an issue about this https://github.com/Microsoft/Recommenders/issues/694. I think we can refactor this ourselves when we change all the notebooks